src/main/java/gameframework/game/GameData.java

```diff
@@ -66,7 +64,13 @@ public GameCanvas getCanvas() {
 	}
 	
 	public void increaseLife(int lifeToAdd) {
-		life.setValue(life.getValue() + lifeToAdd);
+		if (lifeToAdd > 0)	{
+			life.setValue(life.getValue() + lifeToAdd);
+		}
+		else	{
+			System.out.println("Try decreasing instead of addind negative value");
```

Comment:
please throw an `IllegalArgumentException` instead. There is no reason to ignore such a bad API usage